### ISSUE TITLE
Modern: Mobile menu conflicts with WPDownloadManager (bootstrap3)

### ISSUE THREAD

**Issue by eri-trabiccolo** [Nikeo/customizr#1144]
https://wordpress.org/support/topic/menu-bug-with-wpdownloadmanager/

**Comment by eri-trabiccolo** [comment]
a definitive solution would be implement the menu collapsing with the hueman method

**Comment by eri-trabiccolo** [comment]
Update: of course it conflicts also with Lifter LMS plugin.
https://secure.helpscout.net/conversation/446573546/33161?folderId=607709

With the classic style there's no conflict because we changed the 2.3.2 bootstrap js to avoid conflicts.
We really should get rid of the bs collapse js (and the html/css that triggers it) that we use only for the mobile menu.

Again the hueaman method might be the solution, otherwise we can always end up changing our bs 4 collapse js using a different namespace.

@nikeo ^

**Comment by eri-trabiccolo** [comment]
Something similar here:
https://secure.helpscout.net/conversation/475403011/34900?folderId=607709

Actually here the issue is pretty simple.
The plugin's bootstrap js collapse overrides our one.
In bs 3 the collapse active elements had the `in` class added, while in bs 4 they have` show`.
Plus the plugin "scopes" its bootstrap CSS (but not the js), so what happens is the following:
1) clicking on the mobile button, the menu uncollapse
2) due to the bs3 code executed the `in` class is added
3) our CSS code displays a collapsing element when it also has the `show` class => so the collapsing element will still be hidden (display: none)
4) the plugin's CSS code doesn't display the `collapse.in` element because of the scoping: they will display only collapsing elements in this form `.scope .collapse.in`

So basically we should be able to augment the compatibility with that kind of plugin by adding the following rule:
```
.collapse.in {
    display: block;
}
```

(the plugin in question here should be WP-ImmoMakler PLUS - even its free version doesn't seem to be hosted on wp.org, unless I missed it)
